xml/decoders/zimo/PaneAccelDecel.xml

```diff
@@ -69,26 +65,18 @@
         <label>
             <text>A 'Hundreds' digit of 1 offsets the mid value for 'coreless' motors</text>
             <text xml:lang="it">Numero Centinaia =1 è valore intermedio per motori 'coreless'</text>
-            <text xml:lang="cs">Stovky jsou pro coreless motory, 1 je střední hodnota</text>
-            <text xml:lang="de">Der Hunderterwert modifiziert Einstellungen für Glockenanker-Motoren.</text>
```

Comment:
Did you intend to remove all these translations?

Reply:
This is a bit complicated. Originally, I wanted to customize this file for the v37 and provide it with the translation. When uploading the changed file but I always get the message that there is a conflict and this can only be resolved by a user with write permissions. Because I did not find any solution, I decided to leave the original file and create a new one (PaneAccelDecel_alternative.xml) for my purposes. Therefore also the two commits "restore original ...". The file PaneAccelDecel.xml in my branch should therefore be identical to the one in the master. It comes from the test release 4.15.5 and has no German and no Czech translation included.